deploy/helm/sumologic/values.yaml

```
@@ -122,12 +122,12 @@ sumologic:
       default:
         name: logs
         config-name: endpoint-logs
-        # properties can be used to extend default settings, such as processing rules, fields etc
-        #properties:
-          #filters:
-            #- name: "Test Exclude Debug"
-            #filter_type: "Exclude"
-            #regexp: ".*DEBUG.*"
+       # properties can be used to extend default settings, such as processing rules, fields etc
+       # properties:
+         # filters:
+           # - name: "Test Exclude Debug"
+           #   filter_type: "Exclude"
+           #   regexp: ".*DEBUG.*"
```

Good call!
But I like the idea of keeping this in line better, can we do it like the above suggestion?
```suggestion
        
        ## Properties can be used to extend default settings, such as processing rules, fields etc
        #properties:
        #  filters:
        #    - name: "Test Exclude Debug"
        #      filter_type: "Exclude"
        #      regexp: ".*DEBUG.*"

```

---

what about `#properties:` ?

---

`#properties:` is a root key for this whole block